pycoal/mineral.py

```diff
@@ -2,132 +2,219 @@
 
 import spectral
 import pickle
+import numpy
 
 # define number of layers in normalized pixel
 NUMBER_OF_LAYERS = 128
 
-# define wavelength range
-WAVELENGTH_MIN = ???
-WAVELENGTH_MAX = ???
+# define wavelength range in micrometers
+WAVELENGTH_MIN = 0.38315
+WAVELENGTH_MAX = 2.5082
+
+class MineralClassification:
+    def __init__(self):
+        # maybe we could put the constants above in here?
+        # any other variables to move in here?
```

Comment:
Mmmm... if these constants are going to be used elsewhere, then I would prefer they were stored in a separate file and merely imported whereever they were needed. @browtayl do you need to use any of the above in your current work?